tempodb/encoding/vparquet3/schema.go

```diff
@@ -473,6 +480,29 @@ func traceToParquet(meta *backend.BlockMeta, id common.ID, tr *tempopb.Trace, ot
 		}
 	}
 
+	// Calculate service meta information/statistics per trace
+	ot.ServicesMeta = []ServiceMeta{}
+	// TODO: set upper limit on maximum services per trace, to reduce potential overhead when a trace traverses 1000s of services
```

Comment:
+1

Reply:
done

Reply:
I've set `MaxServiceStats = 100`. Any thoughts of increasing or decreasing that number?

I guess 100 catches pretty much all regular traces, and exotic traces with 1000s of services won't be processed, avoiding slowing down the processing of huge traces due to this new feature.

Reply:
I talked with @joe-elliott about this limit today. We concluded, that even for large traces with 1000s of services, the size for the service stats would still be acceptable. It would also be valuable if the service stats were guaranteed to be correct. We are therefore leaning towards removing the `MaxServiceStats` entirely. What do you think?

Reply:
I agree, having incorrect (empty) ServiceStats for some traces is also odd behavior. If you both conclude that the size of the service stats field is not an issue, I'm +1 on removing this limit altogether. I've updated the PR.